reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,76 @@
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
+* VHAL enhancements:
+    - Configuration returned by the external APIs have a new field containing a name and a list of area names for the standard VHAL properties.<!--AC-2290 and AC-2289-->
+    - Property values returned by the external APIs have a new field containing a list of area names for the standard VHAL properties.<!--AC-2290-->
+    - You can replace the Anbox Cloud VHAL implementation with your custom implementation for AAOS images. See [How to use a custom VHAL implementation](tbd). <!--AC-2371-->
+
+### Streaming stack improvements
+
+* Streaming can now be enabled when launching an instance using the `--enable-streaming` option. See [How to stream applications](https://anbox-cloud.io/docs/howto/application/stream) and [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create).<!--Ac-2460 and AC-2360-->
+* VirGL is the default renderer for NVIDIA GPUs with driver version 545 and later. For driver versions earlier than 545, EmuGL is used as the default renderer. See [Rendering architecture](https://anbox-cloud.io/docs/explanation/rendering-architecture).<!--AC-2459, AC-2270 and AC-2348-->
+* Support for [pointer lock](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API) is available with this release. This helps in locking the pointer to a particular browser element, for example, your game window, and limits movement outside the game until the lock is disabled. The pointer lock property must be explicitly enabled on the stream object via the [streaming SDK](https://github.com/canonical/anbox-streaming-sdk).<!--AC-2379-->
+* All components of Anbox Cloud upgrade to NVIDIA 550 driver series by default. <!--AC-2309-->
+
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Starting instances with *Error* status from the web dashboard is supported now.<!--LP 2064899-->
+* Page level guided tours are now available for help in navigating through the dashboard.
+* Support for using custom authentication server instead of Ubuntu SSO.
+* Custom instance name field in *Create instance* form.
+* Enhancements to VHAL panel.
+* Enhancements to forms to support AOSP vs AAOS applications.
+
+
+### Other
+
+* Support for building custom images of Anbox Cloud based on AAOS. See [Custom images](tbd).<!--AC-2458-->
+* Android security updates for May 2024 (see [Android Security Bulletin - May 2024](https://source.android.com/docs/security/bulletin/2024-05-01) for more information).<!--AC-2437-->
+* The Android WebView has been updated to [124.0.6367.82](https://chromereleases.googleblog.com/2024/04/chrome-for-android-update_24.html).
+
+## Removed functionality
+
+* Documentation on how to enable VirGL manually is removed because with this release, VirGL is the default renderer for NVIDIA GPUs.
+
+
+## Known issues
+
```

Comment:
Thanks, I have added these points. 

>  The Anbox Cloud Appliance is being reworked and the current implementation is deprecated. Existing installations of the current implementation are receiving updates until Anbox Cloud 1.24.2. New installations starting with 1.23 will receive the new implementation.

I think it would be good if we give a hint about what the remodel of appliance is about, how it will benefit the users etc. WDYT? If you agree, could you let me know the details?

Reply:
What about: The new implementation of the appliance will simplify the installation process and reduce overall footprint to allow a more simplified operation in different contexts. In detail the new version will no longer use the Juju charmed operators internally but package all necessary service components directly within the snap.